soc/nordic/nrf54l/Kconfig

```
@@ -84,4 +84,10 @@ config SOC_NRF_FORCE_CONSTLAT
 	  of base resources on while in sleep. The advantage of having a constant
 	  and predictable latency will be at the cost of having increased power consumption.
 
+config SOC_NRF54L_ERRATA_56_WORKAROUND
```

Errata is a list of anomalies and workarounds for them, so this word does not really work well in this option name.
```suggestion
config SOC_NRF54L_ANOMALY_56_WORKAROUND
```
See also: https://github.com/zephyrproject-rtos/zephyr/blob/ad4c3e3e9afe1385ab80adf3fa2d3309fad68c60/soc/nordic/nrf53/Kconfig#L46